packages/orbit-components/src/InputSelect/index.tsx

```diff
@@ -351,7 +355,7 @@ const InputSelect = React.forwardRef<HTMLInputElement, Props>(
                   setTopOffset(ev.currentTarget.scrollTop);
                 }
               }}
-              mobileHeader={false}
+              mobileHeader
```

Comment:
@DSil why you enabled this? :) I turned off it on purpose, it should not be like that, should be sticky as it was with input, that's why I have that wrapper.

Reply:
I enabled to have the close button on the top right corner, according to designs. How can it be achieved?

Reply:
As talked IRL, close button should be added manually 👍